crates/sui-move-build/src/lib.rs

```diff
@@ -215,6 +220,17 @@ impl BuildConfig {
     }
 }
 
+fn report_linter_warnings(report: bool) -> anyhow::Result<()> {
+    if report {
+        if let Err(err) = std::io::stderr().write_all(
+            b"Please report feedback on the linter warnings at https://forums.sui.io\n\n",
```

Comment:
Pointing people at the forum seems reasonable to me, but I am obviously open to suggestions on what the message could/should be.